### PR TITLE
[KSP Update] RealHeat KSP 1.3 updates

### PR DESCRIPTION
**Pull request by PhineasFreak**
**Change Log:**

* Add a simple GitHub readme.
* Update the version checker.
* Update the assembly version.
* Update the project **.userprefs** file (was pointing to the Deadly Reentry assemblies).

**Notes:**

* The **String** parameter was converted into lowercase **string**.

### CODE REVIEW

File: Source/Checkers.cs [hunk 1]
```diff
@@ -41,6 +41,7 @@ namespace RealHeat
      */
 
     [KSPAddon(KSPAddon.Startup.Instantly, true)]
+
```
Comment (blowfishpro) [inline Source/Checkers.cs]: Any particular reason for the added newline here?

File: Source/Checkers.cs [hunk 2]
```diff
@@ -84,7 +85,8 @@ public static bool IsUnityCompatible()
         }
 
         // Version of the compatibility checker itself.
-        private static int _version = 5;
+
```
Comment (blowfishpro) [inline Source/Checkers.cs]: and here?

File: Source/Checkers.cs (hunk 1 above)
Comment (PhineasFreak) [inline Source/Checkers.cs]: Probably a "mis-newline". Will fix.

File: Source/Checkers.cs (hunk 2 above)
Comment (PhineasFreak) [inline Source/Checkers.cs]: Just to separate the var from the function for easier readability. I can supress it if you want.

Comment (blowfishpro) [inline Source/Checkers.cs]: Isn't it already separated though?  This newline is between the comment and the field.  It just seemed odd to me.

Comment (PhineasFreak) [inline Source/Checkers.cs]: OK, will also fix.

**Comment by DRVeyl** [comment]
PhineasFreak, the binary in your RealHeat-RH-KSP-1.3-Binaries branch is v4.3, ie an older build.  Would it make sense to compile & commit here, or post via dropbox?

**Comment by PhineasFreak** [comment]
Yes, i am aware of that. I was expecting to make the actual binary after everything settles down and RSS has an official release for KSP 1.3.

**Comment by PhineasFreak** [comment]
**Unofficial** RH assembly (for testing purposes only): https://github.com/PhineasFreak/DocBin/blob/master/DocBin/dll_files/RealHeat.dll

**Comment by eggrobin** [comment]
@PhineasFreak, this pull request needs to update the expected KSP version in Checkers.cs. Further, it needs some changes to calls to the popup functions, whose signatures changed between 1.3.0 and 1.3.1, and the binary needs to be rebuilt for 1.3.1.

With the above, LGTM.

**Comment by PhineasFreak** [comment]
But i updated the version.minor field to "3". The RH checker does not seem to care about the revision number so everything should be OK. Did i miss something else?

Also, forgive my ignorance, but what signatures do you mean? If you mean the new PopupDialog "titleExtra" field then i have updated the code accordingly.

As far as recompiles go, i was hoping that someone else would do it as part of the new release (since the repository itself never included the actual RH .dll assembly, only the code). If adding the .dll file to the repository is not a problem, then i guess that i could go ahead and do so.

**Comment by NathanKell** [comment]
I'll rebuild after the merge. Thanks for this @PhineasFreak !! ❤️

**Comment by eggrobin** [comment]
@PhineasFreak, looking at my git log on my testing branch, it seems I forgot to actually patch in this pull request before testing, hence the nonsensical comments... :-p

**Comment by PhineasFreak** [comment]
@eggrobin Heh, no problem! I had this happen to me before, so i can relate! :smile:

**Pull request merged**